cvat/apps/dataset_manager/formats/mot.py

```diff
@@ -25,34 +25,39 @@ def _import_to_task(dataset, instance_data):
         frame_number = instance_data.abs_frame_id(frame_number)
 
         for ann in item.annotations:
-            if ann.type != datumaro.AnnotationType.bbox:
+            if ann.type != dm.AnnotationType.bbox:
                 continue
 
-            track_id = ann.attributes.get('track_id')
+            occluded = dm.util.cast(ann.attributes.pop('occluded', None), bool) is True
```

Comment:
`bool` casting can be too wide - e.g. `bool('hello')` is `True`. I'd suggest to compare with the expected value directly, i.e. `pop(..., False) is True` instead.

Reply:
Fixed